protos/organization_transaction.proto

```diff
@@ -0,0 +1,96 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+import "organization.proto";
+
+message CreateOrganizationAction {
+	
+	// The name of the organizaiton
+	string name = 1;
+
+	// Type of the organization
+	Organization.OrganizationType organization_type = 2;
+
+	// The industry of the organization
+	string industry = 3;
+
+	// A code representing the identity of the organization
```

Comment:
Is this always the ticker symbol? If yes, the comment could reflect that, for clarity.